### PR TITLE
ROC-5317: Hearing exceptional circumstances

### PR DESCRIPTION
**Pull request by KiranVarma02**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5317

### Change description
Updates to hearing exceptional circumstances as per prototype

### Work checklist

- [x] Unit tests added where applicable
- [x] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/directions-questionnaire/views/hearing-exceptional-circumstances.njk [hunk 1]
```diff
@@ -5,36 +5,76 @@
...
+            radioOption(
+              form = form,
+              label = 'No',
+              name = 'exceptionalCircumstances',
+              value = YesNoOption.NO.option,
+              checked = form.valueFor('exceptionalCircumstances').option === YesNoOption.NO.option,
+              bold = true)
+            }}
+            <div class="form-group panel panel-border-narrow js-hidden " id="exceptionalCircumstances-yes" aria-hidden="false">
+              {{ textArea(label = t('Briefly explain your reasons'), form = form, name = 'reason', inputClass = 'form-control-3-4') }}
```
Comment (robertparkinson) [inline src/main/features/directions-questionnaire/views/hearing-exceptional-circumstances.njk]: This text doesn't need translating

File: src/main/app/court-finder/court.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Address } from './address'
 export class Court {
   constructor (
     public readonly name: string,
-    public readonly distance: number,
+    public readonly slug: string,
```
Comment (robertparkinson) [inline src/main/app/court-finder/court.ts]: slug?

Comment (KiranVarma02) [reply src/main/app/court-finder/court.ts]: yes slug :), don't know why its called that but its something like this `    slug: 'birmingham-district-probate-registry',
`

File: src/main/features/directions-questionnaire/views/hearing-exceptional-circumstances.njk (hunk 1 above)
Comment (KiranVarma02) [reply src/main/features/directions-questionnaire/views/hearing-exceptional-circumstances.njk]: done

File: src/main/app/court-finder/court.ts (hunk 2 above)
Comment (ellie-harrison) [reply src/main/app/court-finder/court.ts]: It's the "Slugified" version of the court name

**Pull request merged**